src/Polysemy/Error.hs

```diff
@@ -242,7 +243,7 @@ mapError f = interpretH $ \case
 {-# INLINE mapError #-}
 
 
-newtype WrappedExc e = WrappedExc { unwrapExc :: e }
+data WrappedExc e = WrappedExc { getExcUid :: {-# UNPACK #-} !Unique, unwrapExc :: e }
```

Comment:
Scrap the type argument. Use `GHC.Exts.Any` instead and `unsafeCoerce` as necessary, like `OpaqueExc` of in-other-words

Reply:
What's the reason for this?

Reply:
Ah, to drop the Typeable constraint? Didn't see the comments below. In what way is the `Typeable` constraint bad?

Reply:
It's no as much that it's bad as it is completely unnecessary, and unnecessary things are bad. It pollutes constraints, making using it just that little bit more harder to use. As the catching logic can be completely driven by `Unique`s, `Typeable` has just become clutter.